presto-main/src/test/java/com/facebook/presto/block/AbstractTestBlock.java

```diff
@@ -224,6 +224,7 @@ private void assertBlockSize(Block block)
         long expectedSecondHalfSize = copyBlockViaBlockSerde(secondHalf).getSizeInBytes();
         assertEquals(secondHalf.getSizeInBytes(), expectedSecondHalfSize);
         assertEquals(block.getRegionSizeInBytes(firstHalf.getPositionCount(), secondHalf.getPositionCount()), expectedSecondHalfSize);
+        assertEquals(block.getSizeInBytes(), block.getRegionSizeInBytes(0, block.getPositionCount()));
```

Comment:
This did not testing `getPositionsSizeInBytes`. The test case you added here is guaranteed by transitive law with line 215 and line 216. My ask is to validate that:

* `getPositionsSizeInBytes([true, true, ...., true])` matches `expectedBlockSize`
* `getPositionsSizeInBytes([true, true, ..., true, false, false, ..., false])` matches `expectedFirstHalfSize`
* similarly for `expectedSecondHalfSize`

Reply:
Oh I didn't see that. So it's already testing that when `getSizeInBytes` is calculated through `getPositionsSizeInBytes` the value is the same as `getRegionSizeInBytes`.